pkg/services/policer/process.go

```diff
@@ -91,7 +91,7 @@ func (p *Policer) poolCapacityWorker(ctx context.Context) {
 
 			if p.taskPool.Cap() != newCapacity {
 				p.taskPool.Tune(newCapacity)
-				p.log.Debug("tune replication capacity",
+				p.log.Info("tune replication capacity",
```

Comment:
Once per second is too frequent for `info`.

Reply:
i may suggest a metric then. want to check this tunning's values and not sure it can help in any scenarios _consistently_

Reply:
agree that 1s logs are annoying, how do we go here?

Reply:
Leave it as debug? Is this behavior configurable?

Reply:
> Leave it as debug?

ok, reverted.

> Is this behavior configurable?

max value with this PR -- yes, but the way it tunes itself in [1, `maxCapacity`] is random to me